frontend/src/components/Auth/Register/Register.js

```diff
@@ -33,7 +33,7 @@ const Register = () => {
   return (
     <div className="Auth">
       <div className="card">
-        <h1>Sudoku Battle Royale</h1>
+        <h1 style={{ color: "white" }}>Sudoku Battle Royale</h1>
```

Comment:
It is better to keep one convention of styling. Mixing inline-styles and css classes will quickly become very hard to maintain. I would suggest exporting this style to a css file and adding a class here (even if it is only this one property).

Reply:
it's only temporary. In this place will be logo image.